test/browser-providers.js

```diff
@@ -2,81 +2,66 @@
 
 /*
  * Browser Configuration for the different jobs in the CI.
- * Target can be either: BS (Browserstack) | SL (Saucelabs) | TC (Travis CI) | null (To not run)
+ *
+ *   - local: Launches the browser locally on the current operating system.
+ *   - BS: Launches the browser within BrowserStack
+ *   - SL: Launches the browser within Saucelabs
+ *
+ * TODO(devversion): rename this to "browserstack" and "saucelabs".
```

Comment:
I've already renamed this to `browserstack` and `saucelabs`. I somewhere have a PR on Material that removes this (just not merged yet).

You can remove this todo.